woocommerce-product-dependencies.php

```diff
@@ -112,11 +112,12 @@ public function woo_tied_products_write_panel() {
 				<p class="form-field">
 					<label>
 						<?php _e( 'Product Dependencies', 'woocommerce-product-dependencies' ); ?>
+						<?php _e( '(Use ID numbers to search for variations)', 'woocommerce-product-dependencies' ); ?>
```

Comment:
This is a bit too verbose for my liking - perhaps you could use something like this to add a tooltip that is backwards compatible with WC versions before 2.5:

```
	/**
	 * Display a WooCommerce help tip.
	 *
	 * @param  string  $tip
	 * @return string
	 */
	public static function wc_help_tip( $tip ) {

		if ( version_compare( WC()->version, '2.5', '>=' ) ) {
			return wc_help_tip( $tip );
		} else {
			return '<img class="help_tip woocommerce-help-tip" data-tip="' . $tip . '" src="' . WC()->plugin_url() . '/assets/images/help.png" />';
		}
	}
```